tools/create-spec

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
```

Comment:
Please add standard license/copyright blurb and a short description.

Reply:
Sure